packages/dart_frog_cli/e2e/test/dev_test.dart

```diff
@@ -47,4 +47,69 @@ void main() {
       expect(response.body, 'Route not found');
     });
   });
+
+  group('dart_frog dev when ran multiple times', () {
+    const projectName1 = 'example1';
+    const projectName2 = 'example2';
+
+    final tempDirectory = Directory.systemTemp.createTempSync();
+
+    Process? process1;
+    Process? process2;
+
+    setUp(() async {
+      await dartFrogCreate(projectName: projectName1, directory: tempDirectory);
+      await dartFrogCreate(projectName: projectName2, directory: tempDirectory);
+    });
+
+    tearDown(() async {
+      if (process1 != null) {
+        killDartFrogServer(process1!.pid).ignore();
```

Comment:
Can we avoid `.ignore()`?

Reply:
If i do it fails to stop the process and throws an exception

```
00:22 +3 -1: dart_frog dev when ran multiple times Should Run 2 different dart_frog dev servers without any problem [E]                                
  Exception: pkill -f dart_frog exited with code 1
  e2e/test/helpers/kill_dart_frog_server.dart 36:7  killDartFrogServer
  ===== asynchronous gap ===========================
  e2e/test/dev_test.dart 71:9     
```